.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,14 @@
+---
+name: 💬 Discussion, questions and/or Help
+about: You need help using Warehouse or want to discuss a topic or feature.
+labels: "Type: Question"
+---
+
+GitHub Issues in `godaddy/warehouse` repository are used exclusively for tracking bugs in Warehouse.
+Please do not submit support requests through GitHub.
```

Comment:
You may want to clarify the difference between a support request and a bug report, since you're directing people to different channels for each.